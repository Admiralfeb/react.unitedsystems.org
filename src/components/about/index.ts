export * from './aboutAllies';
export * from './aboutHC';
export * from './aboutRules';
export * from './carriers';
export * from './about';
