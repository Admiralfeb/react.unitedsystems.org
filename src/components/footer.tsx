import Typography from '@material-ui/core/Typography';
import { Button, makeStyles } from '@material-ui/core';
import Container from '@material-ui/core/Container';
import Link from '@material-ui/core/Link';
import { ReactNode, useContext } from 'react';
import { RealmAppContext } from 'providers';
import { LoginProviders } from 'models/loginProviders';

function ImgCopyright() {
  return (
    <Typography
      variant="body2"
      color="textSecondary"
      style={{ textAlign: 'center' }}
    >
      Thank you to CMDR RaZ uryel for the background image
    </Typography>
  );
}
function Copyright() {
  return (
    <Typography
      variant="body2"
      color="textSecondary"
      style={{ textAlign: 'center' }}
    >
      {'Copyright Â© '}
      <Link color="inherit" href="https://unitedsystems.org/">
        United Systems Cooperative
      </Link>{' '}
      {new Date().getFullYear()}
      {'.'}
    </Typography>
  );
}

const useStyles = makeStyles((theme) => ({
  root: {
    display: 'flex',
    flexDirection: 'column',
    minHeight: '100vh',
  },
  main: {
    marginTop: theme.spacing(1),
    marginBottom: theme.spacing(2),
  },
  footer: {
    padding: theme.spacing(3, 2),
    marginTop: 'auto',
    backgroundColor:
      theme.palette.type === 'light'
        ? theme.palette.grey[200]
        : theme.palette.grey[800],
  },
  center: {
    textAlign: 'center',
  },
}));

export const Footer = (props: { children?: ReactNode | JSX.Element }) => {
  const classes = useStyles();
  const realm = useContext(RealmAppContext);
  const { children } = props;

  return (
    <div className={classes.root}>
      <div className={classes.main}>{children}</div>
      <footer className={classes.footer}>
        <Container maxWidth="sm" className={classes.center}>
          {realm?.checkUserProvider(LoginProviders.userpass) === true && (
            <Button onClick={() => realm.logOut()}>Log out</Button>
          )}
          <Copyright />
          <ImgCopyright />
        </Container>
      </footer>
    </div>
  );
};
