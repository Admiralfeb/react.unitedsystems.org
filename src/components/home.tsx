import { Fade, makeStyles, Paper, Typography } from '@material-ui/core';
import { useDownloadLink } from 'hooks/useDownloadLink';
import {
  lore,
  latin,
  latinTranslation,
  videoID,
  videoPosterID,
} from 'data/home';

const useStyles = makeStyles((theme) => ({
  root: {},
  header: {
    textAlign: 'center',
  },
  paper: {
    textAlign: 'center',
    width: '80%',
    margin: 'auto',
    padding: 5,
    paddingBottom: 10,
    marginBottom: 5,
  },
  latin: {
    display: 'flex',
    flexDirection: 'column',
    textAlign: 'center',
    border: '1px solid white',
    borderRadius: 10,
    width: 350,
    margin: 'auto',
    marginBottom: 10,
  },
  cursive: {
    fontFamily: 'Cinzel, serif',
    fontSize: 32,
  },
  subtitle: {
    flex: '0 0',
  },
  video: {
    width: '100%',
    padding: 0,
  },
  lore: {
    marginBottom: theme.spacing(1),
  },
}));

export const Home = () => {
  const classes = useStyles();
  const { getDownloadLink } = useDownloadLink();
  return (
    <Fade in={true}>
      <div className={classes.root}>
        <Typography variant="h3" className={classes.header}>
          United Systems Cooperative
        </Typography>
        <Paper className={classes.paper}>
          <Typography className={classes.lore}>{lore}</Typography>
          <div className={classes.latin}>
            <Typography className={classes.cursive} variant="h4">
              {latin}
            </Typography>
            <Typography variant="subtitle2" className={classes.subtitle}>
              {latinTranslation}
            </Typography>
          </div>
          <video
            controls
            className={classes.video}
            poster={getDownloadLink(videoPosterID)}
            controlsList="nodownload"
          >
            <source src={getDownloadLink(videoID)} type="video/mp4" />
            "Your browser does not support this video"
          </video>
        </Paper>
      </div>
    </Fade>
  );
};
