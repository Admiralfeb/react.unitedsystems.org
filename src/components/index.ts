export * from './home';
export * from '../routes/information.routes';
export * from './join';
export * from './joinForm';
export * from './notFound';
