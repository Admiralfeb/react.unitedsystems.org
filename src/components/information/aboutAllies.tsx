import {
  List,
  ListItem,
  ListItemText,
  makeStyles,
  Paper,
  Typography,
} from '@material-ui/core';
import { allies } from 'data/information';

const useStyles = makeStyles({
  header: {
    textAlign: 'center',
  },
  allies: {
    border: '3px solid white',
    borderRadius: 10,
    margin: 'auto',
    width: 450,
  },
});

/** Displays Allies */
export const AboutAllies = () => {
  const classes = useStyles();
  return (
    <div>
      <Typography variant="h3" className={classes.header}>
        Allies
      </Typography>
      <Paper className={classes.allies}>
        <List>
          {allies.map((rule: string, i: number) => (
            <ListItem key={i}>
              <ListItemText primary={`${rule}`} />
            </ListItem>
          ))}
        </List>
      </Paper>
    </div>
  );
};
