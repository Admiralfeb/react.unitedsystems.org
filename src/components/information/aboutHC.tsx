import {
  List,
  ListItem,
  ListItemText,
  makeStyles,
  Paper,
  Typography,
} from '@material-ui/core';
import { hc } from 'data/information';

const useStyles = makeStyles({
  root: {},
  header: {
    textAlign: 'center',
  },
  rank: {
    border: '3px solid white',
    borderRadius: 10,
    margin: 'auto',
    width: 450,
  },
});

/** Displays High Command Members */
export const AboutHC = () => {
  const classes = useStyles();
  return (
    <div>
      <Typography variant="h3" className={classes.header}>
        High Command
      </Typography>
      <Paper className={classes.rank}>
        <List>
          {hc.map((rule: string, i: number) => (
            <ListItem key={i}>
              <ListItemText primary={`${rule}`} />
            </ListItem>
          ))}
        </List>
      </Paper>
    </div>
  );
};
