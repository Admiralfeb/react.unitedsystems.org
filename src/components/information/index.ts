export * from './aboutAllies';
export * from './aboutHC';
export * from './aboutRules';
export * from './carriers';
export * from './infoSection';
export * from './infographic';
export * from './information';
export * from './miningMap';
export * from './shipReviews';
