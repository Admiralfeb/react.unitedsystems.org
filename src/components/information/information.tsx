import { makeStyles } from '@material-ui/core';
import { Typography, Paper } from '@material-ui/core';
import { InfoSection } from './infoSection';
import { useInfoButtons } from 'hooks/information/useInfoButtons';

const useStyles = makeStyles((theme) => ({
  header: {
    textAlign: 'center',
  },
  paper: {
    textAlign: 'center',
    width: '80%',
    margin: 'auto',
    padding: 5,
    paddingBottom: 10,
    marginBottom: 5,
  },
  specialButton: {
    display: 'flex',
    flexDirection: 'column',
  },
  secondary: {
    color: theme.palette.secondary.main,
  },
}));

/** Information Landing Component */
export const Information = () => {
  const classes = useStyles();
  const { toolsList, docsList, guidesList, uscLinksList } = useInfoButtons();
  return (
    <>
      <Typography variant="h2" className={classes.header}>
        USC Data Archive
      </Typography>
      <InfoSection
        id="usc-links"
        key="usc-links"
        header="USC Links"
        buttons={uscLinksList}
      />
      <Paper className={classes.paper}>
        <Typography variant="subtitle1">
          New Players look to the{' '}
          <span className={classes.secondary}> blue buttons </span>for helpful
          tips in getting started with the Guides, Tools, and Documentation
          below.
        </Typography>
      </Paper>
      <InfoSection
        id="guides"
        key="guides"
        header="Guides"
        buttons={guidesList}
      />
      <InfoSection id="tools" key="tools" header="Tools" buttons={toolsList} />
      <InfoSection
        id="docs"
        key="docs"
        header="Documentation"
        buttons={docsList}
      />
    </>
  );
};
