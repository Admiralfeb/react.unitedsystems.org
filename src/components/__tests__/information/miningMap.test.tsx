import { render } from '@testing-library/react';
import { MiningMap } from 'components/information';

describe('Mining Maps', () => {
  it('should match the snapshot', () => {
    const { baseElement } = render(<MiningMap />);
    expect(baseElement).toMatchSnapshot();
  });
});
