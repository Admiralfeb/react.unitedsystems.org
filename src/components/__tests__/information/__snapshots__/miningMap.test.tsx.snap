// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mining Maps should match the snapshot 1`] = `
<body>
  <div>
    <div
      class="makeStyles-root-1"
    >
      <h3
        class="MuiTypography-root MuiTypography-h3"
      >
        Mining Maps
      </h3>
      <h6
        class="MuiTypography-root MuiTypography-subtitle1"
      >
        Compiled by Luisqa
      </h6>
      <div
        class="MuiPaper-root MuiTableContainer-root MuiPaper-elevation1 MuiPaper-rounded"
      >
        <table
          class="MuiTable-root"
        >
          <thead
            class="MuiTableHead-root"
          >
            <tr
              class="MuiTableRow-root MuiTableRow-head"
            >
              <th
                class="MuiTableCell-root MuiTableCell-head"
                scope="col"
              >
                System
              </th>
              <th
                class="MuiTableCell-root MuiTableCell-head"
                scope="col"
              >
                Body
              </th>
              <th
                class="MuiTableCell-root MuiTableCell-head"
                scope="col"
              >
                Material
              </th>
              <th
                class="MuiTableCell-root MuiTableCell-head"
                scope="col"
              >
                Mining Type
              </th>
              <th
                class="MuiTableCell-root MuiTableCell-head"
                scope="col"
              >
                Rez Overlap
              </th>
              <th
                class="MuiTableCell-root MuiTableCell-head"
                scope="col"
              >
                Link
              </th>
            </tr>
          </thead>
          <tbody
            class="MuiTableBody-root"
          >
            <tr
              class="MuiTableRow-root"
            >
              <td
                class="MuiTableCell-root MuiTableCell-body"
              >
                Col 285 Sector KM-V d2-106
                 
                <button
                  class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorSecondary MuiIconButton-sizeSmall"
                  tabindex="0"
                  type="button"
                >
                  <span
                    class="MuiIconButton-label"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M16 1H4c-1.1 0-2 .9-2 2v14h2V3h12V1zm-1 4l6 6v10c0 1.1-.9 2-2 2H7.99C6.89 23 6 22.1 6 21l.01-14c0-1.1.89-2 1.99-2h7zm-1 7h5.5L14 6.5V12z"
                      />
                    </svg>
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </button>
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body"
              >
                5 Ring A
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body"
              >
                <a
                  class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
                  href="https://inara.cz/galaxy-commodity/81"
                  target="_blank"
                >
                  Platinum
                </a>
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body"
              >
                Surface Laser
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body"
              />
              <td
                class="MuiTableCell-root MuiTableCell-body"
              >
                <a
                  aria-disabled="false"
                  class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
                  href="https://docs.google.com/document/d/1IQ_izr_kqtKkEUWrVkVjdKoDq3cHw7NZyaZZRY5p60s/edit"
                  tabindex="0"
                  target="_blank"
                >
                  <span
                    class="MuiButton-label"
                  >
                    Open Map
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </a>
              </td>
            </tr>
            <tr
              class="MuiTableRow-root"
            >
              <td
                class="MuiTableCell-root MuiTableCell-body"
              >
                HIP 21991
                 
                <button
                  class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorSecondary MuiIconButton-sizeSmall"
                  tabindex="0"
                  type="button"
                >
                  <span
                    class="MuiIconButton-label"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M16 1H4c-1.1 0-2 .9-2 2v14h2V3h12V1zm-1 4l6 6v10c0 1.1-.9 2-2 2H7.99C6.89 23 6 22.1 6 21l.01-14c0-1.1.89-2 1.99-2h7zm-1 7h5.5L14 6.5V12z"
                      />
                    </svg>
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </button>
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body"
              >
                2
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body"
              >
                <a
                  class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
                  href="https://inara.cz/galaxy-commodity/84"
                  target="_blank"
                >
                  Painite
                </a>
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body"
              >
                Surface Laser
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body"
              >
                High
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body"
              >
                <a
                  aria-disabled="false"
                  class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
                  href="https://docs.google.com/document/d/1CqLij-LZ4Yog_nkE2hVGBUll951RBHLXdj5lt57l2yc/edit"
                  tabindex="0"
                  target="_blank"
                >
                  <span
                    class="MuiButton-label"
                  >
                    Open Map
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </a>
              </td>
            </tr>
            <tr
              class="MuiTableRow-root"
            >
              <td
                class="MuiTableCell-root MuiTableCell-body"
              >
                HIP 35219
                 
                <button
                  class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorSecondary MuiIconButton-sizeSmall"
                  tabindex="0"
                  type="button"
                >
                  <span
                    class="MuiIconButton-label"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M16 1H4c-1.1 0-2 .9-2 2v14h2V3h12V1zm-1 4l6 6v10c0 1.1-.9 2-2 2H7.99C6.89 23 6 22.1 6 21l.01-14c0-1.1.89-2 1.99-2h7zm-1 7h5.5L14 6.5V12z"
                      />
                    </svg>
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </button>
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body"
              >
                2
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body"
              >
                <a
                  class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
                  href="https://inara.cz/galaxy-commodity/84"
                  target="_blank"
                >
                  Painite
                </a>
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body"
              >
                Surface Laser
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body"
              >
                Hazardous
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body"
              >
                <a
                  aria-disabled="false"
                  class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
                  href="https://docs.google.com/document/d/1DmJwuwOQYuLB8_-KZSfqwaKmU7aDv0ZR6Mhmmaiz2ag/edit"
                  tabindex="0"
                  target="_blank"
                >
                  <span
                    class="MuiButton-label"
                  >
                    Open Map
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </a>
              </td>
            </tr>
            <tr
              class="MuiTableRow-root"
            >
              <td
                class="MuiTableCell-root MuiTableCell-body"
              >
                HIP 60953
                 
                <button
                  class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorSecondary MuiIconButton-sizeSmall"
                  tabindex="0"
                  type="button"
                >
                  <span
                    class="MuiIconButton-label"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M16 1H4c-1.1 0-2 .9-2 2v14h2V3h12V1zm-1 4l6 6v10c0 1.1-.9 2-2 2H7.99C6.89 23 6 22.1 6 21l.01-14c0-1.1.89-2 1.99-2h7zm-1 7h5.5L14 6.5V12z"
                      />
                    </svg>
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </button>
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body"
              >
                ABC 8
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body"
              >
                <a
                  class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
                  href="https://inara.cz/galaxy-commodity/84"
                  target="_blank"
                >
                  Painite
                </a>
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body"
              >
                Surface Laser
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body"
              >
                Hazardous
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body"
              >
                <a
                  aria-disabled="false"
                  class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
                  href="https://imgur.com/gallery/VJuICjZ"
                  tabindex="0"
                  target="_blank"
                >
                  <span
                    class="MuiButton-label"
                  >
                    Open Map
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </a>
              </td>
            </tr>
            <tr
              class="MuiTableRow-root"
            >
              <td
                class="MuiTableCell-root MuiTableCell-body"
              >
                Hyades Sector DB-X d1-112
                 
                <button
                  class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorSecondary MuiIconButton-sizeSmall"
                  tabindex="0"
                  type="button"
                >
                  <span
                    class="MuiIconButton-label"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M16 1H4c-1.1 0-2 .9-2 2v14h2V3h12V1zm-1 4l6 6v10c0 1.1-.9 2-2 2H7.99C6.89 23 6 22.1 6 21l.01-14c0-1.1.89-2 1.99-2h7zm-1 7h5.5L14 6.5V12z"
                      />
                    </svg>
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </button>
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body"
              >
                2 Ring A
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body"
              >
                <a
                  class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
                  href="https://inara.cz/galaxy-commodity/84"
                  target="_blank"
                >
                  Painite
                </a>
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body"
              >
                Surface Laser
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body"
              />
              <td
                class="MuiTableCell-root MuiTableCell-body"
              >
                <a
                  aria-disabled="false"
                  class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
                  href="https://docs.google.com/document/d/1cuy00PplPzpd0mgJoZE7O1nQiprTXgz8zf0PGoLn0Cs/edit"
                  tabindex="0"
                  target="_blank"
                >
                  <span
                    class="MuiButton-label"
                  >
                    Open Map
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </a>
              </td>
            </tr>
            <tr
              class="MuiTableRow-root"
            >
              <td
                class="MuiTableCell-root MuiTableCell-body"
              >
                Lalande 34968
                 
                <button
                  class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorSecondary MuiIconButton-sizeSmall"
                  tabindex="0"
                  type="button"
                >
                  <span
                    class="MuiIconButton-label"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M16 1H4c-1.1 0-2 .9-2 2v14h2V3h12V1zm-1 4l6 6v10c0 1.1-.9 2-2 2H7.99C6.89 23 6 22.1 6 21l.01-14c0-1.1.89-2 1.99-2h7zm-1 7h5.5L14 6.5V12z"
                      />
                    </svg>
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </button>
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body"
              >
                AB 8
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body"
              >
                <a
                  class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
                  href="https://inara.cz/galaxy-commodity/81"
                  target="_blank"
                >
                  Platinum
                </a>
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body"
              >
                Surface Laser
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body"
              >
                Hazardous
              </td>
              <td
                class="MuiTableCell-root MuiTableCell-body"
              >
                <a
                  aria-disabled="false"
                  class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
                  href="https://docs.google.com/document/d/1okp9xLc_TsSleibSONaK1q8oGXxMMUm4wJmRm2ZbyPM/edit"
                  tabindex="0"
                  target="_blank"
                >
                  <span
                    class="MuiButton-label"
                  >
                    Open Map
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </a>
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
  </div>
</body>
`;
